pkgs/build-support/cc-wrapper/gnat-wrapper.sh

```diff
@@ -4,11 +4,11 @@ if [ -n "@coreutils_bin@" ]; then
   PATH="@coreutils_bin@/bin"
 fi
 
-if [ -n "$NIX_GNAT_WRAPPER_START_HOOK" ]; then
-    source "$NIX_GNAT_WRAPPER_START_HOOK"
+if [ -n "$NIX_@infixSalt@_GNAT_WRAPPER_START_HOOK" ]; then
+    source "$NIX_@infixSalt@_GNAT_WRAPPER_START_HOOK"
```

Comment:
There are many occurrences of "source", but "." is much more portable. 
Example usage: `. ./myfile.sh`